comictaggerlib/cli.py

```diff
@@ -639,7 +639,7 @@ def save(self, ca: ComicArchive, match_results: OnlineMatchResults) -> tuple[Res
                             month=ct_md.month,
                             year=ct_md.year,
                             publisher=None,
-                            image_url=ct_md._cover_image or "",
+                            image_url=str(ct_md._cover_image) or "",
```

Comment:
Is this okay as is or should it be `ct_md._cover_image if isinstance(issue._cover_image, str) else ""` as well?